# dist_mat callback

---
custom_msgs/matrix[] dist_mat
custom_msgs/axis[] num_name
